sample-compose/app/build.gradle.kts

```diff
@@ -108,8 +108,10 @@ android {
         }
         unitTests.all {
             if (it.name != "testStagingDebugUnitTest") {
-                it.extensions.configure(kotlinx.kover.api.KoverTaskExtension::class) {
-                    isDisabled.set(true)
+                kover {
+                    excludeTests {
+                        tasks(it.name)
+                    }
                 }
             }
         }
```

Comment:
[This approach is deprecated](https://github.com/Kotlin/kotlinx-kover/blob/b35990f1a922cb793977b9627f2a7586fa3533bc/README.md#configuring-jvm-test-task) and has been replaced with `mergeWith("stagingDebug")` from the default `koverReport` config as below.